colpali_engine/models/idefics3/colidefics3/modeling_colidefics3.py

```diff
@@ -1,13 +1,24 @@
+from typing import Optional
+
 from torch import nn
 from transformers import Idefics3Model, Idefics3PreTrainedModel
 
 
 class ColIdefics3(Idefics3PreTrainedModel):
-    def __init__(self, config):
+    def __init__(self, config, remove_context_embeddings: Optional[bool] = False):
```

Comment:
btw this is not an Optional - Optional defaults to None, this is just a kwarg with a default value